### DISCUSSION TITLE
Mention upgrade with Terraform/OpenTofu provider for Flux in release notes

### DISCUSSION THREAD

**Discussion by mloskot** [fluxcd/flux2#5072] [Proposals] [answered]
I noticed, Flux release notes like [this for 2.4.0](https://github.com/fluxcd/flux2/releases/tag/v2.4.0) include the note on upgrade procedure:

> Upgrade Flux from v2.3.0 to v2.4.0 either by [rerunning bootstrap](https://fluxcd.io/flux/installation/#bootstrap-upgrade) or by using the [Flux GitHub Action](https://github.com/fluxcd/flux2/tree/main/action).

Would it be possible to also mention the https://github.com/fluxcd/terraform-provider-flux/ as an additional viable upgrade option?

I have been using the provider for Flux for quite some time now and successfully performed numerous upgrades, and easily.
I think, it is quite important to make it clear to the provider users that they can use it to upgrade the Flux deployment.

**Comment by stefanprodan** [comment, marked as answer]
We should link to the upgrade docs here which captures all options including Terraform https://fluxcd.io/flux/installation/upgrade/

**Reply by mloskot** [reply]
@stefanprodan That would do indeed. I thought of submitting PR updating this section 
https://github.com/fluxcd/flux2/blob/5b83111dc6b3732dd77c714c6a2ad867373c2337/docs/release/release-notes-template.md?plain=1#L46-L49

but I feel like this is too internal content to be modified by someone from outside like myself. I don't even know if this template is still in use.

**Reply by stefanprodan** [reply]
It’s usually me that writes the release notes. You can open a PR for the template, thanks!

**Reply by mloskot** [reply]
Done in https://github.com/fluxcd/flux2/pull/5073 Thank you!
